cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -34,7 +36,7 @@ class YamlConfig(
         val testMethods: List<String> = listOf(),
         val limitBreak: Boolean = false,
         val projectId: String = getDefaultProjectId(),
-        var testShardChunks: List<List<String>> = emptyList()) {
+        var testShardChunks: Set<Set<String>> = emptySet()) {
```

Comment:
`) {` in new line?

Reply:
That's the end of the class constructor. I'm matching the style here:
https://kotlinlang.org/docs/reference/classes.html